packages/patternfly-4/react-core/src/components/Checkbox/examples/UncontrolledCheckbox.js

```diff
@@ -4,7 +4,7 @@ import { Checkbox } from '@patternfly/react-core';
 class UncontrolledCheckbox extends React.Component {
   static title = 'Uncontrolled Checkbox';
   render() {
-    return <Checkbox aria-label="uncontrolled checkbox example" />;
+    return <Checkbox label="Uncontrolled CheckBox" aria-label="uncontrolled checkbox example" id="check-2" />;
```

Comment:
I only see simple strings passed in as labels for the examples. Maybe add another example to pass in custom component